src/wp-includes/script-loader.php

```diff
@@ -1877,7 +1895,16 @@ function wp_print_delayed_inline_script_loader() {
       newScript.type = "text/javascript";
       scripts[i].parentNode.replaceChild(newScript, scripts[i]);
     }
-  };
+  }
+  document.addEventListener("load", onScriptLoad, true);
```

Comment:
The third argument is `useCapture` or `options`.

Reply:
Instead of `true`, this could equivalently be `{capture: true}`.

Reply:
I opted for `useCapture` instead of `options` here since fewer bytes and it has to be passed into `addEventListener` as well as `removeEventListener`.